### PR TITLE
Student list now links to student profile

### PR DESCRIPTION
**Pull request by edavidsonsawyer**
# Who is this PR for?
Administrators using the attendance dashbord

# What problem does this PR fix?
In #1264 @alexsoble suggested adding links to the student profiles of the students in the attendance list. This would allow admins to quickly get contact info for a student

# What does this PR do?
Adds links to student profile from dashboard

# Screenshot (if adding a client-side feature)

# Checklist

+ [X] Tested latest version in an Internet Explorer virtual machine? *(If the PR adds Javascript.)*

### CODE REVIEW

**Comment by edavidsonsawyer** [comment]
@alexsoble @snoopyuri Adding links to the student list was pretty simple, so if we decide that's a sensible feature (I vote yes), we can merge this any time.

File: app/assets/javascripts/school_administrator_dashboard/students_table.jsx [hunk 1]
```diff
@@ -91,10 +96,13 @@ export default React.createClass({
           <tbody>
             {this.orderedRows().map(student => {
               return (
-                <tr>
-                  <td>{student.last_name}</td>
-                  <td>{student.first_name}</td>
-                  <td>{student.absences}</td>
+                <tr key={student.id}>
+                  <td width="66.66%">
```
Comment (alexsoble) [inline app/assets/javascripts/school_administrator_dashboard/students_table.jsx]: This is fine, just as a side note we use `flex` for this in other places in the app.

**Comment by alexsoble** [comment]
@edavidsonsawyer Great! I like this feature/PR because it follows the pattern we set in the school overview of student list items being links.

**Comment by alexsoble** [comment]
Looks great in IE, let's 🚢 .

**Comment by alexsoble** [comment]
@edavidsonsawyer Looks good in prod mi amigo.

**Pull request merged**